app/src/main/java/io/intrepid/contest/screens/contestcreation/ValidatableView.java

```diff
@@ -1,6 +1,6 @@
 package io.intrepid.contest.screens.contestcreation;
 
-public interface ValidatableView {
+public interface ValidatableView extends ContestCreationFragment {
```

Comment:
If we're still extending contest creation fragment here let's move the name back to ValidatableContestCreationFragment just for clarity.